charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -42,6 +42,10 @@ spec:
         {{- end }}
         - -v
         - "{{ .Values.controllerManager.verbosity }}"
+        {{- if .Values.controllerManager.resyncInterval }}
```

Comment:
In fact... this might actually be a problem... because the default value of `5m` isn't truthy. Does the chart actually install clean like this?